app/models/FileInformation.scala

```diff
@@ -34,8 +33,11 @@ case class Metadata(items: Seq[MetadataItem]) {
 object FileInformation {
   implicit val metadataItemReads: Reads[MetadataItem] =
     ((JsPath \ "metadata").read[String] and (JsPath \ "value").read[String]) (MetadataItem.apply _)
+
   implicit val metadataReads: Reads[Metadata] = __.read[List[MetadataItem]].map(Metadata.apply)
+
   implicit val metadataItemWrites: Writes[MetadataItem] =  Json.writes[MetadataItem]
+
   implicit val metadataWrites: Writes[Metadata] = new Writes[Metadata] {
```

Comment:
It could be refactored as below and add a line break before implicit val fileInformationFormats:

  implicit val metadataWrites: Writes[Metadata] = (o: Metadata) => JsArray(o.items.map(
    item => Json.obj(("metadata", item.key), ("value", item.value))))